tests/Functional/Table/Synapse/SynapseTableQueryBuilderTest.php

```diff
@@ -339,7 +339,8 @@ public function testGetDropCommand(): void
         $this->assertNotNull($ref->getObjectId());
 
         $ref = $this->getSynapseTableReflection();
-        $this->expectException(ReflectionException::class);
+        $this->expectException(TableNotExistsReflectionException::class);
+        $this->expectExceptionMessage('Table utils-test_qb-schema.utils-test_test does not exist.');
```

Comment:
Když to tady vidím v testu, tak máme "nepsané pravidlo" psát okolo substituovaných věcí v error message uvozovky. 

```suggestion
        $this->expectExceptionMessage('Table "utils-test_qb-schema.utils-test_test" does not exist.');
```